lib/dfe/reference_data/degrees/grades.rb

```diff
@@ -26,66 +28,77 @@ module Degrees
           'e2fe18d4-8655-47cf-ab1a-8c3e0b0f078f' =>
           { hesa_code: '02',
             name: 'Upper second-class honours (2:1)',
+            abbreviation: '2:1',
             suggestion_synonyms: [],
             match_synonyms: ['Upper second class honours', '2:1', '2.1'],
             group: :main_undergrad },
           '377a46ea-d6c6-4e87-9728-c1f0dd0ef109' =>
           { hesa_code: '03',
             name: 'Lower second-class honours (2:2)',
+            abbreviation: '2:2',
```

Comment:
Can this also be changed to 'Lower second' so it's consistent with the other abbreviations